src/components/displayAllComponent.js

```diff
@@ -20,18 +20,18 @@ class DisplayAllComponent extends Component {
     for (var setting of this.props.reduxState.svgSettingList) {
       if (this.props.reduxState.svgOptions.length > 0) {
         this.props.reduxState.displayOptimize.push({
-          original: setting.dataurl,
+          normalSvg: setting.dataurl,
```

Comment:
normal, in my opinion is not a better name than origional, but adding Svg to it would make it better, perhaps try origionalSvg